client/app/components/dynamic-form/DynamicForm.jsx

```diff
@@ -47,13 +47,13 @@ export class DynamicForm extends React.Component {
   }
 
   setActionInProgress = (actionName, inProgress) => {
-    this.setState({
+    this.setState(prevState => ({
```

Comment:
Cool, I didn't know `prevState` existed here :sweat_smile:

Reply:
Me too. I just read the docs :smile:

Reply:
BTW - it's the same warning as @ranbena mentioned above - `react/no-access-state-in-setstate`

Reply:
Also, `eslint` is smart enough to detect code like this (the same rule):
```javascript
updateCounter() {
  const counter = this.state.counter + 1;
  this.setState({ counter });
}
```
(I tried to modify code this way, but warning didn't disappear :sweat_smile:)